scripts/setup.js

```diff
@@ -0,0 +1,84 @@
+import prompt from 'prompt'
+import {exec} from 'child_process'
+import {writeFile} from 'fs'
+import pkgJson from 'pkg-json'
+
+const onErr = err => console.error(err)
+
+function logCmd (error) {
+  if (error !== null) {
+    onErr(error)
+  }
+}
+
+exec('rimraf ./.git', logCmd)
+
+process.stdin.resume()
+process.stdin.setEncoding('utf-8')
+
+console.log('\n\n\nPlease follow the prompts to setup your new package\n')
+
+prompt.start()
+
+const properties = {
+  packageName: {
+    name: 'packageName',
+    description: 'Enter your package\'s name (@most/)'
+  },
+  description: {
+    name: 'description',
+    description: 'Describe your awesome new package'
+  },
+  fullName: {
+    name: 'fullName',
+    description: 'What is your full name?'
+  },
+  githubUserName: {
+    name: 'githubUserName',
+    description: 'What is your Github username?'
+  },
+  email: {
+    name: 'email',
+    description: 'What is your email?'
+  }
+}
+
+const set = (field, value) => pkgJson({set: field, value})
+
+function handleResults ({packageName, description, fullName, githubUserName, email}) {
+  set('name', `@most/${packageName}`)
+  set('main', `dist/${packageName}.js`)
+  set('files', [`dist/${packageName}.js`])
+  set('scripts.build-dist', `mkdir -p dist && rollup src/index.js | babel --presets es2015 --plugins transform-es2015-modules-umd --module-id '@most/${packageName}' -o dist/${packageName}.js`)
+  set('scripts.build', `npm run build-dist && uglifyjs dist/${packageName}.js -o dist/${packageName}.min.js`)
```

Comment:
Should this also *remove* the `start` script from package.json?  Is there any reason to keep it after running?  Hmmm, maybe if you ctrl-C the script half way through and want to re-run it.  Maybe it could remove the `start` script as the very last thing it does?

Thoughts?